site.yml.sample

```diff
@@ -13,18 +13,6 @@
 # Note that this is just an example which holds for this playbook, tendrl roles
 # doesn't care how you group your servers in ansible inventory.
 
-#
-# Run yum update.
-#
-
-- hosts: all
```

Comment:
Previously, we expected that admin will edit the main playbook file so that he will notice what's going on there, commenting out or tweaking stuff to fit his needs. But this approach didn't work. So now, we no longer expect admin to look into the file and tweak it - which is what this pull request is about, and removal of "run yum update" task was done here as it's not obvious we are doing it in the playbook and some people may not want to run full system update during Tendrl installation.